android-sdk/src/main/java/com/microsoft/hsg/android/HealthVaultFileSettings.java

```diff
@@ -36,37 +36,32 @@
  */
 public class HealthVaultFileSettings implements HealthVaultSettings {
 
-    /** The properties. */
-    private Properties properties;
-    
-    /** The ctx. */
-    private Context ctx;
-    
-    /**
-     * Instantiates a new health vault file settings.
-     * 
-     * @param ctx the ctx
-     */
-    public HealthVaultFileSettings(Context ctx)
-    {
-        this.ctx = ctx;
-        
-        try
-        {
-            properties = new Properties();
-            try
-            {
-                properties.load(ctx.openFileInput("settings.props"));
-            }
-            catch(FileNotFoundException fnfe)
-            {
-            }
-        }
-        catch(IOException ioe)
-        {
-            throw new HVSystemException("Could not load properties.", ioe);
-        }
-    }
+	/** The properties. */
+	private Properties properties;
+
+	/** The ctx. */
+	private Context ctx;
+
+	/**
+	 * Instantiates a new health vault file settings.
+	 *
+	 * @param ctx the ctx
+	 */
+	public HealthVaultFileSettings(Context ctx) {
+		this.ctx = ctx;
+
+		try {
+			properties = new Properties();
+			try {
+				properties.load(ctx.openFileInput("settings.props"));
```

Comment:
openFileInput returns an input stream which needs to be closed, not sure if properties.load will handle that.